README.md

```diff
@@ -337,6 +337,64 @@ are done with the migration.
 
   Modify this column if desired.
 
+### `partitions` ###
+
+- `schema` (type `name`): schema of the table with the partition
```

Comment:
How about "schema of the partitioned table"?

Reply:
Changed in 6b6497ebd7036d27fac47843743abc787086c51c

Reply:
&#x2713;